### PR TITLE
December 2019 news

### PR DESCRIPTION
**Pull request by aeslaughter**
@idaholab/moose-team Please add your work from Nov.

(refs #11446)

### CODE REVIEW

**Comment by permcody** [comment]
@aeslaughter or @milljm - One of you should probably put something in here about Python3. Last month was when we went live.

**Comment by bwspenc** [comment]
@aeslaughter I would like to send out an announcement somewhere about the timeline for removing the solid_mechanics module. Would that make sense in the newsletter?  It's not a new feature, but it's an important announcement, and this might be the best place to get the word out to everyone (in addition to a message to the users list).

**Comment by lindsayad** [comment]
I think that seems appropriate here

**Comment by permcody** [comment]
@bwspenc - Please push a commit on top of this PR and we'll happily include it.

File: modules/doc/content/newsletter/2019_12.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+
+We have been publicizing that this change has been coming for a long time, and expect that any new models that are being set up now are based on the `tensor_mechanics` module. The BISON team has put together an [extensive guide](https://mooseframework.org/bison/tutorials/mechanics_conversion/conversion_guidelines.html) on how to migrate existing models to use `tensor_mechanics` as part of BISON's code documentation. While there are details in that guide that apply only to BISON models, the general principles are applicable to converting any model that uses mechanics.
```
Comment (bwspenc) [inline modules/doc/content/newsletter/2019_12.md]: @sapitts The version that is publicly available is much less complete than the current version (although still better than nothing). Any thoughts on how to handle that?  I'm not sure what the schedule is for updating the website.  Maybe we could move general material somewhere on the MOOSE documentation side?

Comment (sapitts) [reply modules/doc/content/newsletter/2019_12.md]: @gardnerru says there is no schedule and that it used to be that "we were lucky to update it once a year"; the current version is stuck in export control. Perhaps we should wait on putting this announcement about deprecating solid mechanics until January.

We certainly could move the general information and probably add in the few additional classes....this is a depressing unleased project

Comment (permcody) [reply modules/doc/content/newsletter/2019_12.md]: Are you guys happy with this? Should we get this merged?

Comment (sapitts) [reply modules/doc/content/newsletter/2019_12.md]: The more updated migration guide in BISON just passed the review process minutes ago, so we can probably merge this section as written

**Review by permcody — APPROVED** [review]

**Pull request merged**